test/test_viscous.py

```diff
@@ -244,9 +243,11 @@ def inf_norm(x):
         assert inf_norm(heat_flux - xp_heat_flux) < 2e-8
 
         # verify diffusive mass flux is zilch (no scalar components)
-        from mirgecom.viscous import diffusive_flux
-        j = diffusive_flux(discr, eos, cv, grad_cv)
-        assert len(j) == 0
+        # Don't call for non-multi CV
+        if cv.nspecies > 0:
+            from mirgecom.viscous import diffusive_flux
+            j = diffusive_flux(discr, eos, cv, grad_cv)
+            assert len(j) == 0
```

Comment:
This looks strange to me... I think the non-multi case is the only time `len(j)` would be 0, right? @MTCam?

Reply:
You are correct.  This is fallout from the change where j was being carried around as an empty array - but when CV is split - it will no longer make sense to call `diffusive_flux` with a non-multi-component CV.   This entire snippet perhaps should have been removed.

Reply:
Removed: (0824668)